### DISCUSSION TITLE
如何在其他路径生成相应的ide工程

### DISCUSSION THREAD

**Discussion by supf1994** [xmake-io/xmake#2677] [Q&A]
### 你在什么场景下需要该功能？

目前在vs和mingw下测试跨平台程序，需要再不同的平台下分别用不同的工具链生成构建，使用 xmake project -k xmakefile  或xmake project -k vsxmake，不知道该如何将相应的makefile或vs工程生成在不同的目录

### 描述可能的解决方案

是否有相应的功能

**Comment by waruqi** [comment]
干嘛不直接构建，非得要生成。直接 `xmake` 编译不好么、

xmake project -k vsxmake outputdir
